Sources/SwiftProtobuf/TextFormatScanner.swift

```diff
@@ -720,6 +722,28 @@ internal struct TextFormatScanner {
         return d
     }
 
+    private mutating func skipOptionalKeywordCaseSensitive(bytes: [UInt8]) -> Bool {
+        let start = p
+        for b in bytes {
+            if p == end || p[0] != b{
+                p = start
+                return false
+            }
+            p += 1
+        }
+        if p == end {
+            return true
+        }
+        let c = p[0]
+        if ((c >= asciiUpperA && c <= asciiUpperZ)
```

Comment:
Is this check complete enough?  what about a number ("true2you")?  Would it be safer to ensure this is followed by whitespace, a comma, ']', or '}'?  Since that's normally breaks a token in text format?

Reply:
Fixed (and tested).